PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalRecurringBillingDetails.kt

```diff
@@ -35,6 +38,8 @@ data class PayPalRecurringBillingDetails @JvmOverloads constructor(
     var productQuantity: Int? = null,
     var shippingAmount: String? = null,
     var taxAmount: String? = null,
+    val unitAmount: String? = null,
+    val amountBreakdown: RecurringBillingAmountBreakdown? = null,
```

Comment:
Should we make these `var`s since they're optional properties? This would create the Java setter methods.